cpu/native/syscalls.c

```diff
@@ -255,6 +275,12 @@ int putchar(int c)
     return 0;
 }
 
+int putc(int c, FILE *fp)
+{
+    _native_write(fileno(fp), &c, 1);
```

Comment:
```suggestion
    char tmp = c;
    _native_write(fileno(fp), &tmp, 1);
```

Otherwise this would break on big endian systems.

(I heared that everyone should care very much about these, despite nobody actually ever have seen one. Maybe big endian systems are just a big conspirency like Bielefeld, and they are completely made up?)